core/common/src/IModel.ts

```diff
@@ -285,6 +294,18 @@ export class EcefLocation implements EcefLocationProps {
     return new EcefLocation({ origin: ecefOrigin, orientation: YawPitchRollAngles.createFromMatrix3d(matrix)!, cartographicOrigin: origin });
   }
 
+  /** Construct ECEF Location from transform with optional position on the earth used to establish the ECEF origin and orientation. */
+  public static createFromTransform(transform: Transform): EcefLocation {
+    const ecefOrigin = transform.getOrigin();
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    const angleFromInput = YawPitchRollAngles.createDegrees(0,0,0);
```

Comment:
```suggestion
    const angleFromInput = YawPitchRollAngles.createDegrees(0,0,0);
```
`angleFromInput` isn't unused.